rootex/script/interpreter.cpp

```diff
@@ -402,6 +402,7 @@ void LuaInterpreter::registerTypes()
 		scene["snatchChild"] = &Scene::snatchChild;
 		scene["getID"] = &Scene::getID;
 		scene["getFullName"] = &Scene::getFullName;
+		scene["setFullName"] = &Scene::setFullName;
```

Comment:
I don't think allowing the name to be changed is a good idea. afaik it is related to the path + filename.